source/clear-linux/get-started/bare-metal-install-desktop/bare-metal-install-desktop.rst

```diff
@@ -0,0 +1,437 @@
+**********
+
+* Use the :kbd:`Tab` key to navigate :guilabel:`Required options`
+  and :guilabel:`Advanced options`
+
+* Use :kbd:`Up` or :kbd:`Down` arrow keys to navigate submenu list.
+
+* Select :kbd:`Confirm`, or :kbd:`Cancel` in submenus.
+
+Required options
+****************
+
+Select Time Zone
+================
+
+#. From the Main Menu, select :guilabel:`Select Time Zone`. `UTC` is selected
+   by default.
+
+#. In :guilabel:`Select Time Zone`, navigate to the desired time zone.
+   Or start typing the region and then city.
+   (.e.g., :file:`America/Los_Angeles`).
+
+#. Select :guilabel:`Confirm`.
+
+   .. figure:: figures/bare-metal-install-desktop-05.png
+      :scale: 100%
+      :alt: Select System Timezone
+
+      Figure 5: Select System Time Zone
+
+Select Keyboard
+===============
+
+#. From the Main Menu, select :guilabel:`Select Keyboard`.
+
+#. Navigate to your desired keyboard layout. We select "us" for the
+   United States.
+
+#. Select :guilabel:`Confirm`.
+
+   .. figure:: figures/bare-metal-install-desktop-06.png
+      :scale: 100%
+      :alt: Select Keyboard menu
+
+      Figure 6: Select Keyboard menu
+
+Select Installation Media
+=========================
+
+#. From the Main Menu, select :guilabel:`Select Installation Media`.
+
+#. Choose an installation method: `Safe Installation`_ or
+   `Destructive Installation`_.
+
+   .. figure:: figures/bare-metal-install-desktop-07.png
+      :scale: 100%
+      :alt: Select Installation Media
+
+      Figure 7: Select Installation Media
+
+Safe Installation
+-----------------
+Use this method to safely install |CL| on media with available space, or
+alongside existing partitions, and accept the `Default partition schema`_.
+If enough free space exists, safe installation is allowed.
+
+Destructive Installation
+------------------------
+
+Use this method to destroy the contents of the target device, install |CL| on it, and accept the `Default partition schema`_.
+
+Disk encryption
+===============
+
+For greater security, disk encryption is supported using LUKS. Encryption is
+optional.
+
+#. To encrypt the root partition, select :guilabel:`Enable Encryption`,
+   as shown in Figure 8.
+
+   .. figure:: figures/bare-metal-install-desktop-08.png
+      :scale: 100%
+      :alt: Enable Encryption
+
+      Figure 8: Enable Encryption
+
+#. When :guilabel:`Encryption Passphrase` appears, enter a passphrase.
+
+   .. figure:: figures/bare-metal-install-desktop-09.png
+      :scale: 100%
+      :alt: Encryption Passphrase
+
+      Figure 9: Encryption Passphrase
+
+   .. note::
+
+      Minimum length is 8 characters. Maximum length is 94 characters.
+
+#. Enter the same passphrase in the second field.
+
+#. Select :guilabel:`Confirm` in the dialogue box.
+
+   .. note::
+
+      :guilabel:`Confirm` is only highlighted if passphrases match.
+
+#. Select :guilabel:`Confirm` in submenu.
+
+Manage User
+===========
+
+#. In Advanced Options, select :guilabel:`Manage User`.
```

Comment:
Manage User is a **Required** Option, not Advance for the GUI.

Reply:
Good catch. Holdover from `-beta` version. Cheers.